src/main/java/roomescape/controller/TimeController.java

```diff
@@ -0,0 +1,57 @@
+package roomescape.controller;
+
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import roomescape.dto.request.CreateTimeRequest;
+import roomescape.dto.response.TimeResponse;
+import roomescape.service.TimeService;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/times")
+public class TimeController {
+
+    private static final String HEADER_LOCATION = "Location";
+    private static final String LOCATION_DEFAULT_VALUE = "/times/";
+
+    private final TimeService timeService;
+
+    public TimeController(final TimeService timeService) {
+        this.timeService = timeService;
+    }
+
+    @PostMapping
+    public ResponseEntity<TimeResponse> createTime(@RequestBody CreateTimeRequest request) {
+        TimeResponse response = timeService.createTime(request);
+
+        HttpHeaders headers = new HttpHeaders();
+        headers.add(HEADER_LOCATION, LOCATION_DEFAULT_VALUE + response.id());
+
+        return ResponseEntity.status(HttpStatus.CREATED)
+                .headers(headers)
+                .body(response);
```

Comment:
```suggestion
        return ResponseEntity.created(URI.create(LOCATION_DEFAULT_VALUE + response.id()))
                .body(response);
```

1. 아래에 있는 방법을 알고 계셨나요?
2. 둘 중에 어느 방법을 더 선호하시나요?

Reply:
1. 몰랐습니다!
2. 알려주신 방법이 더 좋은 것 같아요.
상수 선언 및 헤더 객체 생성을 안해도 되어서 코드가 더 간결해졌네요ㅎㅎ